### ISSUE TITLE
Nvda does not announce that the edit field can be automatically configured with values when walking with the above and below arrows

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9386]
### Steps to reproduce:
Open a document in word.
Press alt
Arrow right to the layout tab.
tab, and press enter.
You'll be in the top edit field where you can edit the top margin value.
Nvda announces edition that indicates that I can enter the new value.
However in this box we can configure the new values also walking with arrows above and below.
Since nvda only announces an editing field, an unexpected user may think that he should only enter the new desired value, he will not suspect that he can also configure the values with arrows above and below.

### Actual behavior:
For the controls of editing of this window and possible other controls the nvda announces only field of edition
### Expected behavior:
The nvda can announce the edit field followed by another indication that announces to an unexpected user that he can set new values with arrows above and below.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2018.4.1
#### Windows version:
10 17134
#### Name and version of other software in use when reproducing the issue:
Microsoft® Word 2016 MSO (16.0.4266.1001) 

#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no

**Comment by Adriani90** [comment]
Actually this is an edit field with a spin button, you can navigate this with object navigation. This should be reported like for example "margin, spin button edit field 0%" when pressing tab and shift+tab. This is also valid for all other edit fields with a spin button in MS Word. @lukaszgo1 is it possible to improve this in NVDA? I can imagine this is rather not exposed correctly by Microsoft.
cc: @michaelDCurran
